src/kv.rs

```diff
@@ -65,10 +65,10 @@ pub(crate) async fn get_links(store: &Store) -> Result<LinkDefinitionList> {
 pub(crate) async fn put_link(store: &Store, ld: LinkDefinition) -> Result<()> {
     let id = ld_hash(&ld);
     let key = format!("{}{}", LINKDEF_PREFIX, id);
-    store
+    Ok(store
```

Comment:
What was this for?

Reply:
This gave me an error when compiling, something about error types, which seemed easiest to correct like this. Kinda strange